package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "AbsoluteHack",
-  "version": "1.0",
+  "version": "1.0.0",
```

Comment:
Change version to ```3.1.0```

Reply:
I already did.
You'd be having a merge conflict.